android/build.gradle

```diff
@@ -39,10 +53,8 @@ android {
 
 dependencies {
     expectedBy project(':common')
-
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    api "com.github.yandextaxitech:binaryprefs:1.0.1"
-
+    compile "com.github.yandextaxitech:binaryprefs:1.0.1"
```

Comment:
Add comment which will explain why we have to use `compile` instead of `implementation`